src/backend/aspen/app/app.py

```diff
@@ -107,7 +107,17 @@ def validate_auth_header(auth_header):
 
 
 def setup_userinfo(user_id):
-    user = get_usergroup_query(g.db_session, user_id).one()
+    sentry_sdk.set_user(
+        {
+            "requested_user_id": user_id,
+        }
+    )
+    user = get_usergroup_query(g.db_session, user_id).one_or_none()
```

Comment:
Can we leave this as a .one() call inside a try / except NoResultFound block instead? I think it's better to be explicit that getting no rows back is an exception.

Reply:
Done!